nixos/modules/services/web-apps/languagetool.nix

```diff
@@ -0,0 +1,47 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  cfg = config.services.languageToolHttp;
+  format = pkgs.formats.toml { };
+in
+{
+  options.services.languageToolHttp = {
+    enable = mkEnableOption "the LanguagTool http server";
+    allowBrowserPluginAccess = mkEnableOption "access from the browser plugin";
+    settings = mkOption {
+      description = ''
+        Contents of the configuration file.
+      '';
+      inherit (format) type;
+      default = { };
+    };
+    args = mkOption {
```

Comment:
What is the relationship of command line augments to configuration file? Are they configuring the same thing? Is there overlap? When do you use command line arguments vs settings?

Please include link to documentation for possible arguments and their purpose.